cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -99,7 +103,7 @@ public Samples getSamples(Cluster cluster,
           continue;
         }
         long recordTime = record.value().time();
-        if (recordTime < startTimeMs) {
+        if (recordTime + _acceptableMetricRecordProduceDelayMs < startTimeMs) {
```

Comment:
can we update the debug message on the next line to reflect the `acceptableMetricRecordProduceDelay`?